meta/runtime.yml

```diff
@@ -1,3 +1,278 @@
+  - floating_ip
+  - group_assignment
+  - group_assignment
+  - host_aggregate
+  - host_aggregate
+  - identity_domain
+  - identity_domain
+  - identity_domain_info
+  - identity_domain_info
+  - identity_group
+  - identity_group
+  - identity_group_info
+  - identity_group_info
+  - identity_role
+  - identity_role
+  - identity_user
+  - identity_user
+  - identity_user_info
+  - identity_user_info
+  - image
+  - image_info
+  - keypair
+  - keystone_federation_protocol
+  - keystone_federation_protocol_info
+  - lb_listener
+  - lb_listener
+  - lb_member
+  - lb_member
+  - lb_pool
+  - lb_pool
+  - loadbalancer
+  - network
+  - networks_info
+  - object
+  - port
+  - port_info
+  - project
+  - project_access
+  - project_info
+  - quota
+  - recordset
+  - role_assignment
+  - role_assignment
+  - router
+  - routers_info
+  - security_group
+  - security_group_rule
+  - server
+  - server_action
+  - server_group
+  - server_info
+  - server_metadata
+  - server_volume
+  - stack
+  - subnet
+  - subnets_info
+  - volume
+  - volume_snapshot
+  - os_auth
+  - os_client_config
+  - os_client_config
+  - os_coe_cluster
+  - os_coe_cluster_template
+  - os_endpoint
+  - os_flavor
+  - os_flavor_info
+  - os_flavor_info
+  - os_floating_ip
+  - os_group
+  - os_group
+  - os_group_info
+  - os_group_info
+  - os_image
+  - os_image_info
+  - os_ironic
+  - os_ironic
+  - os_ironic_inspect
+  - os_ironic_inspect
+  - os_ironic_node
+  - os_ironic_node
+  - os_keypair
+  - os_keystone_domain
+  - os_keystone_domain
+  - os_keystone_domain_info
+  - os_keystone_domain_info
+  - os_keystone_endpoint
+  - os_keystone_endpoint
+  - os_keystone_federation_protocol
+  - os_keystone_federation_protocol_info
+  - os_keystone_identity_provider
+  - os_keystone_identity_provider
+  - os_keystone_identity_provider_info
+  - os_keystone_identity_provider_info
+  - os_keystone_mapping
+  - os_keystone_mapping
+  - os_keystone_mapping_info
+  - os_keystone_mapping_info
+  - os_keystone_role
+  - os_keystone_role
+  - os_keystone_service
+  - os_keystone_service
+  - os_listener
+  - os_listener
+  - os_loadbalancer
+  - os_member
+  - os_member
+  - os_network
+  - os_networks_info
+  - os_nova_flavor
+  - os_nova_flavor
+  - os_nova_host_aggregate
+  - os_nova_host_aggregate
+  - os_object
+  - os_pool
+  - os_pool
+  - os_port
+  - os_port_info
+  - os_project
+  - os_project_access
+  - os_project_info
+  - os_quota
+  - os_recordset
+  - os_router
+  - os_routers_info
+  - os_security_group
+  - os_security_group_rule
+  - os_server
+  - os_server_action
+  - os_server_group
+  - os_server_info
+  - os_server_metadata
+  - os_server_volume
+  - os_stack
+  - os_subnet
+  - os_subnets_info
+  - os_user
+  - os_user
+  - os_user_group
+  - os_user_group
+  - os_user_info
+  - os_user_info
+  - os_user_role
+  - os_user_role
+  - os_volume
+  - os_volume_snapshot
+  - os_zone
+  - os_zone
+  
+action_groups_redirection:
+  os:
```

Comment:
@s-hertel I have a question about redirect:
should it be instead:
```yaml
action_groups_redirection:
  openstack:
    redirect: os
```
If so, can we just drop here section of `openstack` and leave only `os`? It's actually duplicate each of other.
Or maybe I miss something in redirection?
Thanks

Reply:
The change was in flux still in ansible/ansible at the time, and `action_groups_redirection` needs to be removed from this now.

Reply:
Created patch in Gerrit:
Move action_group to runtime  https://review.opendev.org/735977